platzky_google_tag_manager/plugin.py

```
@@ -0,0 +1,57 @@
+
+class GoogleTagManagerConfig(PluginBaseConfig):
+    """Configuration model for the Google Tag Manager plugin."""
+
+    ID: str
+
+    @field_validator("ID")
+    @classmethod
+    def id_must_not_be_empty(cls, v: str) -> str:
+        """Validate that the GTM ID is not empty."""
+        if not v.strip():
+            raise ValueError("GTM ID must not be empty")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)